src/dist/omero-ms-image-region.service

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=OMERO image region microservice server
+Requires=multi-user.target
 Requires=redis.service
```

Comment:
Should remove this dep with the presence of multi-user.target, it only adds complexity and is incompatible with external redis.

Reply:
Thanks, removed in [b605694](https://github.com/glencoesoftware/omero-ms-image-region/pull/141/commits/b6056941fb0e0e32aa651d32583bef59e8f1084e)